src/components/MultiHopTrade/hooks/useGetTradeQuotes/useGetTradeQuotes.tsx

```diff
@@ -147,6 +159,7 @@ export const useGetTradeQuotes = () => {
           affiliateBps: willDonate ? DEFAULT_SWAPPER_DONATION_BPS : '0',
           // Pass in the user's slippage preference if it's set, else let the swapper use its default
           slippageTolerancePercentage: userSlippageTolerancePercentage,
+          ...maybePubKey,
```

Comment:
same here, the an iife+object+spread seems heavy to populate a single key 🤌
```suggestion
          pubKey: isLedger(wallet) ? fromAccountId(sellAccountId).account : undefined,
```

Reply:
Might as well ✅  12bf1a6e65